team/views.py

```diff
@@ -0,0 +1,151 @@
+from django.http import Http404
+from rest_framework import status
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from .serializers import MemberSerializer
+from .models import Member
+
+
+class MemberView(APIView):
```

Comment:
One of the advantages of a framework like DRF is that it helps developers correctly implement certain aspects of RESTful frameworks, like making sure requests and responses represent the resource.

I think if you read about [Generic API Views and mixins](http://www.django-rest-framework.org/tutorial/3-class-based-views/#using-mixins), you'll find you can make this code shorter and more consistent.

Reply:
Thanks! That was helpful, updating the code.